app/build.gradle

```diff
@@ -54,6 +54,11 @@ android {
         // debug build. This seems to be a Gradle bug, therefore
         // TODO: update Gradle version
         release {
+            if (System.properties.containsKey('packageSuffix')) {
+                applicationIdSuffix System.getProperty('packageSuffix')
```

Comment:
I would pass the `packageSuffix` parameter through the same sanitization as the git branch name

Reply:
I would say that this is not needed.  
In the git branch case the name of something unrelated (branch name) is added to the package id, app_name and archive name. And because the branch name allows for a wider range of values, it needs to be sanitized.

In this case the provided parameter is exclusivly used for those 3 things. Additionally I would rather like to know that my `packageSuffix` is not valid than get an app with the wrong name

Reply:
Yes, it's not needed then